CONTRIBUTING.md

```diff
@@ -1,123 +1,55 @@
+### Make a fork
 You should be familiar with the basics of
 [contributing on GitHub](https://help.github.com/articles/using-pull-requests) and have a fork
 [properly set up](https://github.com/ohmybash/oh-my-bash/wiki/Contribution-Technical-Practices).
 
 You MUST always create PRs with _a dedicated branch_ based on the latest upstream tree.
 
-If you create your own PR, please make sure you do it right. Also be so kind as to reference
-any issue that would be solved in the PR description body,
-[for instance](https://help.github.com/articles/closing-issues-via-commit-messages/)
-_"Fixes #XXXX"_ for issue number XXXX.
-
-### You have a solution
-
-Please be so kind as to [search](#use-the-search-luke) for any open issue already covering
-your [problem](#you-have-a-problem), and any pending/merged/rejected PR covering your solution.
-
-If the solution is already reported, try it out and +1 the pull request if the
-solution works ok. On the other hand, if you think your solution is better, post
-it with a reference to the other one so we can have both solutions to compare.
-
-If not, then go ahead and submit a PR. Please copy to anyone relevant (e.g. plugin
-maintainers) by mentioning their GitHub handle (starting with `@`) in your message.
-
-### You have an addition
-
-Please [do not](https://github.com/ohmybash/oh-my-bash/wiki/Themes#dont-send-us-your-theme-for-now)
-send themes for now.
-
-Please be so kind as to [search](#use-the-search-luke) for any pending, merged or rejected Pull Requests
-covering or related to what you want to add.
-
-If you find one, try it out and work with the author on a common solution.
-
-If not, then go ahead and submit a PR. Please copy to anyone relevant (_eg_ plugin
-maintainers) by mentioning their GitHub handle (starting with `@`) in your message.
-
-For any extensive change, _eg_ a new plugin, you will have to find testers to +1 your PR.
-
-----
+### Form of merge requests
+All merge requests has to have following naming:
+```
+<FILE>: <Summary>
 
-## Use the Search, Luke
+<Description>
+<Fixes/Bug>: #<BUG_NUMBER>
+Signed-off-by: <name> <surname> <e-mail>
+```
 
-_May the Force (of past experiences) be with you_
+Example:
+```
+theme/agnoster: Fixed naming
 
-GitHub offers [many search features](https://help.github.com/articles/searching-github/)
-to help you check whether a similar contribution to yours already exists. Please search
-before making any contribution, it avoids duplicates and eases maintenance. Trust me,
-that works 90% of the time.
+Agnoster theme is using wrong file naming which is fixed in this commit
+Fixes: #70
+Fixes: #59
+Signed-off-by: Jacob Hrbek <werifgx@gmail.com>
+```
+Where signature is optional.
 
-You can also take a look at the [FAQ](https://github.com/ohmybash/oh-my-bash/wiki/FAQ)
-to be sure your contribution has not already come up.
+### Quality Assurance (QA)
+All merge requests have to pass spellcheck (https://www.shellcheck.net/) unless stated otherwise.
```

Comment:
typo spellcheck -> shellcheck

```suggestion
All merge requests have to pass shellcheck (https://www.shellcheck.net/) unless stated otherwise.
```

The same typos are also found in other lines.